docs/source/app-dev/authorization.rst

```diff
@@ -193,9 +193,9 @@ where all of the fields are optional, and if present,
 - ``exp`` is the standard JWT expiration date (in seconds since EPOCH)
 - ``actAs``, ``readAs`` and (participant) ``admin`` encode the rights granted by this access token
 
-The ``public`` right is implicitly granted to any bearing a valid JWT issued by a trusted issuer (even without being an admin or being able to act or read on behalf of any party).
+The ``public`` right is implicitly granted to any request bearing a non-expired JWT issued by a trusted issuer with matching ``ledgerId``, ``participantId`` and ``applicationId`` values.
```

Comment:
Maybe worth clarifying that these fields are not required. If they’re there they must be correct but not having them is also fine.

Reply:
This is done on paragraph above "all of the fields are optional"